dotnet/test/common/CustomDriverConfigs/StableChannelEdgeDriver.cs

```diff
@@ -14,9 +14,13 @@ public StableChannelEdgeDriver(EdgeOptions options)
         {
         }
 
+        public StableChannelEdgeDriver(EdgeDriverService service, EdgeOptions options)
+            : base(service, options)
+        {
+        }
         public static EdgeOptions DefaultOptions
         {
-            get { return new EdgeOptions(); }
+            get { return new EdgeOptions() { AcceptInsecureCertificates = true }; }
```

Comment:
I don't think we should set it by default. I am returning from vacation soon, will look through deeper.

Reply:
I added it to match Chrome, and these are just the tests, so I'm not sure it matters either way. But if it doesn't matter we should remove it from both. Test should specify anything that doesn't match defaults.